config/api.js

```diff
@@ -0,0 +1,17 @@
+const { create } = require('axios');
+
+const api = create({
+  baseURL: process.env.BASE_API || '',
+  timeout: 1500,
+  headers: {
+    'Content-Type': 'application/json'
+  }
+});
+
+const getData = url =>
```

Comment:
the config files shouldn't have logic, only variable assignment, this could be a service